working-groups/rfc-2229/README.md

```diff
@@ -0,0 +1,42 @@
+# rfc-2229 Working Group
+![working group status: active][status]
+
+- **Leads:** [@blitzerr][Blitzerr], [@nikomatsakis][Niko]
+- **Meeting Notes:** [NOTES]
+- **FAQs:** [All](FAQ.md)
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Finish, optimize, and test the implementation of [RFC-2229].
```

Comment:
Probably a good idea to put a sentence or two about what RFC 2229 does.

Reply:
Something like:

RFC 2229 changes the way closure capture works so that we no longer capture individual variables, instead capturing the actual path that is used. This helps to eliminate borrow check errors.

Reply:
That's a concise description. I will add.